public/tailwindPlugin.js

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable @typescript-eslint/explicit-function-return-type */
 /* eslint-disable @typescript-eslint/no-require-imports */
 /* eslint-disable @typescript-eslint/no-var-requires */
-const forms = require('@tailwindcss/forms')
```

Comment:
I'm not quite understanding why we're removing the forms plugin from the prefect-design plugin. Wont this have negative effects assuming we're depending on the forms plugin always being installed whenever the prefect-design plugin is used?

Reply:
I thought forms was added to the config? That's what I'm seeing locally which appears to be valid looking at the types.

Reply:
It is, yeah. Forms was previously added twice. Once in the tailwindPlugin file, where it was the third argument and I don't think it was actually doing anything, then once in the tailwind.config file, where it was actually being imported and subsequently bundled with the build.

Reply:
Okay, trust you're correct 👍